pytorch_lightning/utilities/seed.py

```diff
@@ -41,18 +41,17 @@ def seed_everything(seed: Optional[int] = None) -> int:
 
     try:
         if seed is None:
-            seed = os.environ.get("PL_GLOBAL_SEED", _select_seed_randomly(min_seed_value, max_seed_value))
```

Comment:
the function here would be executed in any case even if the env variable contains the value, and this would lead to an unexpected message. this is what you try to prevent right? if so, great, makes sense!

Reply:
yes exactly!